install/kubernetes/cilium/values.yaml.tmpl

```
@@ -1012,6 +1012,8 @@ encryption:
     keyRotationDuration: "5m"
     # -- Enable IPsec encrypted overlay
     encryptedOverlay: false
+    # -- Enable Receive Packet Steering (RPS) for ESP packets.
```

If this is experimental, I think we should call this out here, e.g. 
```suggestion
    # -- Enable Receive Packet Steering (RPS) for ESP packets (experimental).
```

---

Good idea, thanks!